crates/core/src/client/constants.rs

```diff
@@ -39,3 +39,22 @@ pub mod gas {
     /// As a result, the priority fee is set to 0.
     pub const MAX_PRIORITY_FEE_PER_GAS: U128 = U128::ZERO;
 }
+
+// This module contains constants which are being used in place of real data that should be fetched
+// in production.
+pub mod mock_evm_data {
+    use reth_primitives::{H256, H64};
+
+    pub const GAS_LIMIT: u64 = 1_000_000;
+    pub const GAS_USED: u64 = 500_000;
+    pub const CUMULATIVE_GAS_USED: u64 = 1_000_000;
+    pub const EFFECTIVE_GAS_PRICE: u64 = 1_000_000;
+    pub const SIZE: u64 = 1_000_000;
+    pub const MAX_FEE: u64 = 100_000_000_000_000_000;
+    pub const ESTIMATE_GAS: u64 = 100_000_000_000_000_000;
+    pub const TRANSACTION_TYPE: u64 = 0;
+    pub const NONCE: H64 = H64::zero();
+    pub const MIX_HASH: H256 = H256::zero();
+    pub const DIFFCIULTY: u64 = 0;
+    pub const TOTAL_DIFFCIULTY: u64 = 0;
```

Comment:
typo,
thinking of setting it as None, always. wdyt

Reply:
I guess you are talking about `DIFFICULTY` and `TOTAL_DIFFICULTY` right?

It makes sense for them to be none, although I would say, we should make a separate issue for that, and fix it in a different PR, it will require changing a little bit in tests as well { we test for Some(U256::ZERO) }.

Reply:
Fine with me, can you create an issue?